index.js

```diff
@@ -58,13 +60,34 @@ function handleRTCMessage(client, payload)
   }
 }
 
+getDisabledDefault = function(field)
```

Comment:
This function is ugly, and only exists to adapt to the fact that the new setting I'v created are strings. I think I'm just going to change them to be two booleans, for each of audio and video:

`* audio_enabled`
`* audio_on_by_default`
`* video_enabled`
`* video_on_by_default`

Which makes the existing `enabled` a bit confusing in light of this. What if we deprecate (continue to support indefinitely) this one in favor of `on_by_default`, `conversation_on_by_default`, etc?

Reply:
Actually, maybe the other way around. `videoTrack.enabled` etc is already in webrtc spec, and it means "enabled right now" rather than site-wide. I need a new word for "disabled site-wide"...

Reply:
How about this:

* `enabled` (existing settign)
* `audio_allowed`
* `audio_enabled_on_start`
* `video_allowed`
* `video_enabled_on_start`

Reply:
Could still alias `enabled` with `enabled_on_start`.